### ISSUE TITLE
Difference between SR and JK flip-flop

### ISSUE THREAD

**Issue by sgpritam** [CircuitVerse/Interactive-Book#86]
Feature request:
There no explanation of difference between SR and JK flip-flop
There no explanation of difference between D and T flip-flop

**Comment by kakubotics** [comment]
> Feature request:
> There no explanation of difference between SR and JK flip-flop
> There no explanation of difference between D and T flip-flop

hi @sgpritam I am taking this feature request...

**Comment by kakubotics** [comment]
Hi @sgpritam, I have made this feature request happen and it's working pretty well.
I have made two new pages for the difference between SR-JK flip-flop and D-T flip-flop.
I believe the issue is resolved now!
![Screenshot from 2020-03-17 11-02-17](https://user-images.githubusercontent.com/22686059/76861194-34da9980-6882-11ea-9985-5222f233c085.png)
![Screenshot from 2020-03-17 11-01-56](https://user-images.githubusercontent.com/22686059/76861201-373cf380-6882-11ea-93de-7b101c7fb470.png)
